src/Synapse/Synapse.Test/ScenarioTests/SqlPoolBackupTests.ps1

```diff
@@ -105,6 +105,77 @@ function Test-SqlPoolRestorePoint
 	}
 }
 
+function Test-SqlPoolGeoBackup
+{
+	# Setup
+	$testSuffix = 'ps2502'
+	$params = Get-SqlPoolBackupTestEnvironmentParameters $testSuffix
+
+	try
+	{	
+	    $SqlPoolGeoBackupGet =Get-AzSynapseSqlPoolGeoBackup -ResourceGroupName $params.rgname -WorkspaceName $params.workspaceName
+        Assert-AreEqual $params.sqlPoolName $SqlPoolGeoBackupGet[0].Name
+        Assert-Null $SqlPoolGeoBackupGet[0].ElasticPoolName
+        Assert-NotNull $SqlPoolGeoBackupGet[0].Edition
+        Assert-NotNull $SqlPoolGeoBackupGet[0].LastAvailableBackupDate
+
+        $SqlPoolGeoBackupGetByPool =Get-AzSynapseSqlPoolGeoBackup -ResourceGroupName $params.rgname -WorkspaceName $params.workspaceName -Name $params.sqlPoolName
+        Assert-AreEqual 1 $SqlPoolGeoBackupGetByPool.Count
+        Assert-AreEqual $params.sqlPoolName $SqlPoolGeoBackupGetByPool.Name
+        Assert-Null $SqlPoolGeoBackupGetByPool.ElasticPoolName
+        Assert-NotNull $SqlPoolGeoBackupGetByPool.Edition
+        Assert-NotNull $SqlPoolGeoBackupGetByPool.LastAvailableBackupDate
+    }
+	finally
+	{
+		# Cleanup
+		#Remove-SqlPoolBackupTestEnvironment $testSuffix
+	}
+}
+
+function Test-DroppedSqlPool
+{
+	# Setup
+	$testSuffix = 'ps2504'
+	$params = Get-SqlPoolBackupTestEnvironmentParameters $testSuffix
+    
+    $rgname = $params.rgname
+    $workspaceName = $params.workspaceName
+    $sqlPoolName = $params.sqlPoolName
+
+	try
+	{	
+        New-AzSynapseSqlPool -ResourceGroupName $rgname -WorkspaceName $workspaceName -Name $sqlPoolName  -PerformanceLevel DW400c
+        Wait-Seconds 80
+        Remove-AzSynapseSqlPool -ResourceGroupName $rgname -WorkspaceName $workspaceName -Name $sqlPoolName -Force
+        Wait-Seconds 90
+
```

Comment:
Is this necessary?

Reply:
I see the point of waiting. Could you please add some comments above the code for the reason of `Wait-Seconds`.

---
In reply to: [561780177](https://github.com/Azure/azure-powershell/pull/13621#discussion_r561780177) [](ancestors = 561780177)